pkgs/applications/video/obs-studio/plugins/obs-freeze-filter.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, obs-studio
+, qtbase
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-freeze-filter";
+  version = "0.3.3";
+  src = fetchFromGitHub {
+    owner = "exeldro";
+    repo = "obs-freeze-filter";
+    rev = version;
+    sha256 = "sha256-CaHBTfdk8VFjmiclG61elj35glQafgz5B4ENo+7J35o=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [
+    obs-studio
+    qtbase
+  ];
+
+  postInstall = ''
+    rm -rf "$out/share"
+    mkdir -p "$out/share/obs"
+    mv "$out/data/obs-plugins" "$out/share/obs"
+    rm -rf "$out/obs-plugins" "$out/data"
+  '';
+
+  dontWrapQtApps = true;
+
+  meta = with lib; {
+    description = "Plugin for OBS Studio to freeze a source using a filter ";
+    homepage = "https://github.com/exeldro/obs-freeze-filter";
+    license = licenses.gpl2;
+    platforms = platforms.linux;
+  };
```

Comment:
Would you like to add yourself as maintainer here?

Reply:
Yes. Done in 86cb069686fbf851e57570094f0badc91dd785da